libraries/rush-lib/src/cli/scriptActions/PhasedScriptAction.ts

```diff
@@ -190,13 +190,18 @@ export class PhasedScriptAction extends BaseScriptAction<IPhasedCommandConfig> {
       });
     }
 
-    this._selectionParameters = new SelectionParameterSet(this.rushConfiguration, this, {
-      // Include lockfile processing since this expands the selection, and we need to select
-      // at least the same projects selected with the same query to "rush build"
-      includeExternalDependencies: true,
-      // Enable filtering to reduce evaluation cost
-      enableFiltering: true
-    });
+    this._selectionParameters = new SelectionParameterSet(
+      this.rushConfiguration,
+      this,
+      {
+        // Include lockfile processing since this expands the selection, and we need to select
+        // at least the same projects selected with the same query to "rush build"
+        includeExternalDependencies: true,
+        // Enable filtering to reduce evaluation cost
+        enableFiltering: true
+      },
+      false
```

Comment:
Why can't someone run `rush build --subspace my-subspace`?

Reply:
They can do `rush build --only subspace:my-subspace`. But that is an "unsafe" selector, because it does not guarantee that dependencies get built.  Thus a convenient `--subspace` macro isn't a good idea.